its/ruling/src/test/resources/expected/python-S2638.json

```diff
@@ -82,6 +85,13 @@
 'project:django-cms-3.7.1/cms/tests/test_permmod.py': [
 665
 ],
+'project:mypy-0.782/mypy/semanal.py': [
+4682,
```

Comment:
This looks like a false positive. In the rule description we say this is okay:
* Extend the ways a parameter can be provided, i.e. change a keyword-only or positional-only parameter to a keyword-or-positional parameter. This is only true if the order of positional parameters doesn’t change. New positional parameters should be placed at the end

In this case it seems that `code` goes from keyword-only to keyword-or-positional with no other change, so it looks like we shouldn't raise the issue.
It doesn't look related to the ticket itself so maybe creating a separate ticket would make sense.